entity-framework/core/get-started/install/index.md

```diff
@@ -1,110 +1,127 @@
 ---
-title: Installing EF Core
+title: Installing Entiy Framework Core
 author: divega
 ms.date: 08/06/2017
 ms.assetid: 608cc774-c570-4809-8a3e-cd2c8446b8b2
 uid: core/get-started/install/index
 ---
-# Installing EF Core
+# Installing Entity Framework Core
 
 ## Prerequisites
 
-In order to develop .NET Core 2.1 applications (including ASP.NET Core 2.1 applications that target .NET Core) you will need to download and install a version of the [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/core) that is appropriate to your platform. **This is true even if you have installed Visual Studio 2017 version 15.7.**
+* To develop apps that target .NET Core 2.1, install [the .NET Core 2.1 SDK](https://www.microsoft.com/net/download/core). The SDK has to be installed even if you have the latest version of Visual Studio 2017.
 
-In order to use EF Core 2.1 or any other .NET Standard 2.0 library with a .NET platform besides .NET Core 2.1 (for example, with .NET Framework 4.6.1 or greater) you will need a version of NuGet that is aware of the .NET Standard 2.0 and its compatible frameworks. Here are a few ways you can obtain this:
+* To use Visual Studio for development of apps that target .NET Core 2.1, install Visual Studio 2017 version 15.7 or later.
 
-* Install Visual Studio 2017 version 15.7
-* If you are using Visual Studio 2015, [download and upgrade NuGet client to version 3.6.0](https://www.nuget.org/downloads)
+* To use Entity Framework 2.1 in ASP.NET Core applications, use ASP.NET Core 2.1. Applications that use earlier versions of ASP.NET Core must be updated to 2.1.
 
-Projects created with previous versions of Visual Studio and targeting .NET Framework may need additional modifications in order to be compatible with .NET Standard 2.0 libraries:
+* You can use Visual Studio 2015 for apps that target the .NET Framework 4.6.1 or later. But you need a version of NuGet that is aware of the .NET Standard 2.0 and its compatible frameworks. To get that in Visual Studio 2015, [upgrade the NuGet client to version 3.6.0](https://www.nuget.org/downloads).
 
-* Edit the project file and make sure the following entry appears in the initial property group:
-  ``` xml
-  <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-  ```
-
-* For test projects, also make sure the following entry is present:
-  ``` xml
-  <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
-  ```
+## Get the Entity Framework Core runtime
 
-## Getting the bits
-The recommended way to add EF Core runtime libraries into an application is to install an EF Core database provider from NuGet.
+To add EF Core runtime libraries to an application, install the NuGet package for the database provider you want to use. For a list of supported providers and their NuGet package names, see [Database providers](../../providers/index.md).
```

Comment:
Add a paragraph mentioning that ASP.NET Core 2.1 applications include some providers, like SQL Server and In memory?